php/Blocks/Loader.php

```diff
@@ -134,7 +134,7 @@ public function editor_assets() {
 		wp_enqueue_script(
 			$js_handle,
 			$this->assets['url']['entry'],
-			[],
```

Comment:
I made the `dependencies` argument `[]` in https://github.com/studiopress/genesis-custom-blocks/commit/c347a98b31a64b726cae0ed74733def0c203a6cb.

That was to prevent this PHP warning on `wp-admin/widgets.php`:

```
Notice: Function wp_enqueue_script() was called incorrectly. "wp-editor" script should not be enqueued together with the new widgets editor (wp-edit-widgets or wp-customize-widgets). Please see Debugging in WordPress for more information. (This message was added in version 5.8.0.) in wp-includes/functions.php on line 6031
```

Reply:
But that warning is less important now. 

Default themes like [Twenty Twentythree](https://github.com/wordpress/twentytwentythree) don't have widgets, and widgets are soft-deprecated in favor of site editing.